docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -599,3 +599,33 @@ Then `${context-width}` is > `0`
 Then `${context-x-coordinate}` is > `0`
 Then `${context-y-coordinate}` is > `0`
 ----
+
+
+=== Gets a list of elements by locator and check that it is sorted by text with using sortingOrder by element text
+
+==== *_Info_*
+
+Gets a list of elements by locator and check that it is sorted by text with using sortingOrder by element text. List should have more than 1 elements.
```

Comment:
`List should have more than 1 elements.` -> `The list should have more than 1 element.`